packages/blade/.storybook/react/preview.js

```diff
@@ -38,12 +39,60 @@ export const parameters = {
   },
   docs: {
     theme,
+    components: {
+      summary: styled.summary`
```

Comment:
If we can override, can we not add Blade's components here?

We can. I did. It was painful. More context in description here - #817